integration/travis.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -ex
+
+sudo groupadd nobody
+sudo useradd hello
```

Comment:
It is, but with the wrong path - the hook created in the integration test calls `/sbin/adduser`; on ubuntu the adduser binary is under `/usr/sbin`. As a result, the hook does nothing and the user doesn't exist unless created earlier. This is one of those moments where clearing PATH in hooks really bites us because these are two different distributions.